setup.sh

```diff
@@ -32,43 +32,6 @@ else
     exit 1
 fi
 
-# macOS Specific Checks
-if [[ $unamestr == 'Darwin' ]]; then
-    export ARCHFLAGS='-arch x86_64'
-    export LDFLAGS='-L/usr/local/opt/openssl/lib'
-    export CFLAGS='-I/usr/local/opt/openssl/include'
-    current_macos_version="$(sw_vers -productVersion | awk -F '.' '{print $1 "." $2}')"
-    major=$(echo "$current_macos_version" | cut -d'.' -f1)
-    # Check if xcode is installed
-    xcode-select -v
```

Comment:
xcode is not required for compiling any of the dependencies?

Reply:
Not on my Mac, will ask friends to test too

Reply:
when you install git commande line is installed , so I revert back the test , but remove all the rest